requirements.in

```diff
@@ -3,20 +3,20 @@
 #    python -m scripts.install_third_party
 #
 # to update `requirements.txt` and `third_party/python_libs` accordingly.
-apache-beam[gcp]==2.35.0
+apache-beam[gcp]==2.36.0
 beautifulsoup4==4.10.0
 bleach==4.1.0
 defusedxml==0.7.1
 elasticsearch==7.10.1
-firebase-admin==5.1.0
+firebase-admin==5.2.0
 future==0.18.2
 googledatastore==7.0.2
-google-cloud-storage==1.42.3
+google-cloud-storage==2.1.0
 google-auth==1.35.0
 google-cloud-dataflow-client==0.3.1
-google-cloud-logging==2.7.0
+google-cloud-logging==3.0.0
```

Comment:
Just to check we don't have to worry about this major upgrade, right?

Reply:
No, the major change is that version 3 does not support Python 2 anymore.